printing/printer.ml

```diff
@@ -838,6 +838,7 @@ type axiom =
   | Constant of Constant.t (* An axiom or a constant. *)
   | Positive of MutInd.t (* A mutually inductive definition which has been assumed positive. *)
   | Guarded of Constant.t (* a constant whose (co)fixpoints have been assumed to be guarded *)
+  | TypeInType of Constant.t (* a constant which relies on type in type *)
```

Comment:
This seems wrong, since inductives may contain arbitrary terms disabling guard or universe checking may affect them too. I guess Guarded and TypeInType should be `of GlobRef.t`, and `assumptions.ml` changed to check those flags for inductives too.